src/NavbarToggler.js

```diff
@@ -31,7 +31,7 @@ const NavbarToggler = (props) => {
   ), cssModule);
 
   return (
-    <Tag {...attributes} className={classes}>
+    <Tag {...attributes} className={classes} aria-label="Toggle navigation">
```

Comment:
I would place it before `{...attributes}` so that the attributes can override it.

Reply:
Makes sense. Done 👍